database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -13,10 +13,15 @@ public function up(): void
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
-            $table->string('email')->unique();
+            $table->string('first_name');
+            $table->string('last_name');
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->string('phone_number');
```

Comment:
make the phone_number birthday city gender as nullable
we can't force the user to provide his birthday in order to complete registeration